linux_os/guide/services/obsolete/tftp/tftpd_secure_directory.var

```diff
@@ -0,0 +1,14 @@
+documentation_complete: true
```

Comment:
usually we have variable files starting with prefix `var_`. What do you think about this?

Reply:
+1

Reply:
I am impartial - those files end with `var` already, and you can't confuse a variable with something else just by accident. But if someone requires to prepend `var_` in order for the PR to get merged, then I wouldn't object.

Reply:
There are lots of inconsistent variables. I suggest that one day we unite it and document it somewhere, currently this works and I would like to move forward. So if you don't mind I would like to leav it as it is.

Reply:
This should be called `var_tftpd_secure_directory.var`. This makes it easier to understand that it is a variable when the content is fully generated into SCAP. The older use of variables is a lot harder to change since they are used in production and would introduce some painful changes.